doc/release-tasklist

```diff
@@ -73,6 +84,7 @@
       gpg --detach-sign --armor manual-$TAG.pdf
       sha1sum aspect-$TAG.tar.gz manual-$TAG.pdf >sha1sum-$TAG.txt
   . create a release on github, upload tar.gz and manual-$VER.pdf
+  . readme / release notes: add zenodo DOI button
```

Comment:
Can you elaborate on this a bit more? I would not know where or how to add a DOI button.

Reply:
yeah. I think this is something we should discuss before we put this into effect. Basically, @ljhwang said we could do something like this: https://github.com/geodynamics/pylith/releases/tag/v2.2.1

Reply:
Seems nice. Are there arguments against this?